swap_meet/decor.py

```diff
@@ -1,2 +1,15 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, id=None, width=0, length=0, condition=0 ):
```

Comment:
[nit] no space needed before the `)`

```suggestion
    def __init__(self, id=None, width=0, length=0, condition=0):
```